Pivot type changes

dshishkov opened the pull request:
- Do not reset expansions when only certain display related fields are changed on the node

decrapifier commented:
<!--
  0 failure: 
  1 warning:  The README has no...
  1 messages
  
-->


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  The README has not been updated. Please update the README.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  We were able to automatically fix some formatting issues in this PR for you!
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

daedalus28 commented on src/utils.js:
```diff
@@ -72,10 +70,13 @@ let runTypeFunction = config => async (name, item, search) => {
+      message: `Failed running search for ${item.type} (${
+        item.key
+      }) at ${name}: ${_.getOr(error, 'message', error)}`,
+      error,
```
We should probably put the actual `item` on here too as `node` to make it really easy to inspect

Akrion replied:
Added item

daedalus28 commented on src/utils.js:
```diff
@@ -72,10 +70,13 @@ let runTypeFunction = config => async (name, item, search) => {
+      message: `Failed running search for ${item.type} (${
+        item.key
+      }) at ${name}: ${_.getOr(error, 'message', error)}`,
+      error: F.extendOn(error, { node: item }),
```
seems like we could just have `message: '', error, item`

dshishkov committed:
Pivot type changes

- Do not reset expansions when only certain display related fields are changed on the node

dshishkov committed:
use changeset

lint-action committed:
Fix code style issues with Prettier

dshishkov committed:
empty line

dshishkov committed:
Merge remote-tracking branch 'origin/feature/pivotResetExpansionFix' into feature/pivotResetExpansionFix

dshishkov committed:
cleanup
